### ISSUE TITLE
rerunning rsconnect_deploy deploys new API 

### ISSUE THREAD

**Issue by gsingh91** [rstudio/vetiver-python#96]
**Describe the bug**
using `rsconnect_deploy` to redeploy an existing API on Connect, it deploys a new API. I did not specify the `new` argument either

**To Reproduce**
Use can use the jupyter notebook published here.

**Expected behavior**
Redeploying an existing API should overwrite the existing content.

**Comment by isabelizimm** [comment]
Ah, the `new` argument has a default of "True," so you can choose to turn that off by setting it equal to False. If you think it is strongly expected to be False, I can either update it or remove the default altogether. 👍

**Comment by juliasilge** [comment]
Is this the argument default?

https://github.com/rstudio/vetiver-python/blob/91c9856fef02ba99bf83bccf39db2dedc4c88aa1/vetiver/rsconnect.py#L16

Or am I missing something?

**Comment by isabelizimm** [comment]
You are correct! I originally had thought there was something else going wrong. Upon further review, it looks like `rsconnect.deploy_python_fastapi` will think that every deployment is "new." When a model is sent to connect with `deploy_rsconnect`, it is deployed from a temporary directory where an `app.py` file is created from a pinned model. This temporary directory holds the manifest JSON file that would normally be used to determine if the deployment was "new" or not. An escape hatch out of this not-actually-new behavior that currently exists is to supply an `app_id`, which will force-overwrite the existing content.

I'll have to peek into `rsconnect` to see what validating newness looks like 👀
